database/insertData.js

```diff
@@ -729,7 +741,9 @@ const queries = [
                       title: "Intro Section 2 - Page 1/2"
                       elementTypePluginCode: "textInfo"
                       category: INFORMATION
-                      parameters: { label: "Company location" }
+                      parameters: {
+                        title: "Company location"
+                      }
```

Comment:
Did you manually re-format this? Can I suggest leaving all formatting of these queries the same as the GraphQL web interface formats them, otherwise we'll have to keep updating them all the time, which will be massive hassle. (ie. next time I change something in this query, I'll copy-paste the whole thing from the web interface and it'll automatically be changed back, so seems futile to change it now)

Reply:
Yes. But I was also fixing a problem here. I'll change to the format as GraphQL would set.

Reply:
Done: [ed12b85](https://github.com/openmsupply/application-manager-server/pull/162/commits/ed12b85ed8d1860e19a31da61e77ef649f2c0a4b)